### PR TITLE
Add docker build

### PR DESCRIPTION
**Pull request by vyruz1986**

### CODE REVIEW

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -2,10 +2,13 @@ name: .NET
 
 on:
   push:
-    branches: [master]
+    branches: [master, add-docker-build]
```
Comment (Joren-Thijs) [inline .github/workflows/dotnet.yml:5]: Is this branch supposed to stay here?

**Comment by vyruz1986** [comment]
fixes #7

**Review by Joren-Thijs — CHANGES_REQUESTED** [review]
Almost perfect?

**Review by Joren-Thijs — APPROVED** [review]
image

**Pull request merged**